### PR TITLE
More affordable powerups

### PR DESCRIPTION
**Pull request by troggy**
Reduced min share from 1/10000 to 1/100000. Added public method to get min babz to power up

### CODE REVIEW

File: contracts/controller/PowerEnabled.sol [hunk 1]
```diff
@@ -125,7 +131,7 @@ contract PowerEnabled is MarketEnabled {
   function createDownRequest(address _owner, uint256 _amountPower) public onlyPower whenNotPaused {
     // prevent powering down tiny amounts
     // when powering down, at least totalSupply/minShare Power should be claimed
-    require(_amountPower >= authorizedPower().div(10000)); // minShare = 10000;
+    require(_amountPower >= authorizedPower().div(MIN_SHARE_OF_POWER)); // minShare = 10000;
```
Comment (troggy) [inline contracts/controller/PowerEnabled.sol]: not sure we should make min power downs smaller as well @johannbarbie

File: contracts/controller/PowerEnabled.sol [hunk 2]
```diff
@@ -34,6 +36,10 @@ contract PowerEnabled is MarketEnabled {
     downtime = _downtime;
   }
 
+  function minimumPowerUpSizeBabz() public constant returns (uint256) {
```
Comment (troggy) [inline contracts/controller/PowerEnabled.sol:39]: could be used on the frontend to show the users the minimum allowed power up

File: contracts/controller/PowerEnabled.sol (hunk 1 above)
Comment (zobroj) [reply contracts/controller/PowerEnabled.sol]: You might want to update/remove the comment on line 134 & 93, as it may be confusing.

Comment (troggy) [reply contracts/controller/PowerEnabled.sol]: good point!

**Review by johba37 — APPROVED** [review]

**Pull request merged**